packages/core/frontend/sirius-components-core/package.json

```diff
@@ -33,18 +33,20 @@
   },
   "peerDependencies": {
     "@apollo/client": "3.10.4",
-    "@material-ui/core": "4.12.4",
-    "@material-ui/icons": "4.11.3",
+    "@mui/icons-material": "5.15.19",
+    "@mui/material": "5.15.19",
     "@xstate/react": "1.6.3",
     "graphql": "16.8.1",
     "react": "17.0.2",
+    "react-dom": "17.0.2",
```

Comment:
Is react-dom really necessary here? Unless I missed something it should be removed

Reply:
After analysis, it's the other way around, `react-dom` is a peer dependency of `@mui/material` so it should be everywhere (which is currently not the case).

Reply:
Done